### ISSUE TITLE
All NCAA 2024 season request come back with errors

### ISSUE THREAD

**Issue by mileskee** [sportsdataverse/hoopR#132]
If I try to get data from the current college basketball season, it returns an error, but the same functions work for past seasons.

**Comment by Coop197823** [comment]
yes, we are waiting on the developer to fix the code to pull in the 2024 data.  You will see a 20224 dataset here (https://github.com/sportsdataverse/sportsdataverse-data/releases/tag/espn_mens_college_basketball_schedules) when it is done.  I Opened an issue last week and he has yet to respond.

**Comment by saiemgilani** [comment]
Try now to confirm

**Comment by aqsmith08** [comment]
@mileskee Would you be willing to share the command that you're running?

When I run `hoopR::load_mbb_schedule(2024)` and `hoopR::load_mbb_player_box(seasons = 2024)`, it loads the appropriate data now.

**Comment by mileskee** [comment]
> hoopR::load_mbb_player_box(seasons = 2024)

Those are working for me now too, I originally tried loading in data from KenPom, which is still not working for me.

**Comment by saiemgilani** [comment]
> > hoopR::load_mbb_player_box(seasons = 2024)
> 
> Those are working for me now too, I originally tried loading in data from KenPom, which is still not working for me.

Seems a separate issue which appears valid, will look into it
